packages/cli/src/cli/basemaps/action.tileset.history.ts

```diff
@@ -94,19 +94,19 @@ export class TileSetHistoryAction extends TileSetBaseAction {
     showDiff(
```

Comment:
somewhat related,  could you make this static, or possibly move it into the utils function, would be helpful to reuse in other places :)